bin/infrasim-init

```diff
@@ -54,20 +55,21 @@ def create_infrasim_directories():
         os.system("rm -rf /usr/local/libexec")
         os.mkdir("/usr/local/libexec")
 
+
 def create_infrasim_conf():
 
     # Prepare default network
     networks = []
     nics_list = netifaces.interfaces()
-    eth_nic = filter(lambda x: 'e' in x,nics_list)[0]
+    eth_nic = filter(lambda x: 'e' in x, nics_list)[0]
     mac = create_mac_address()
     networks.append({"nic": eth_nic, "mac": mac})
 
     # Prepare default disk
     disks = []
     disks.append({"size": 8})
 
-    # Render infrasim.conf
+    # Render infrasim.yml
     infrasim_conf = ""
     with open(INFRASIM_TEMP_CONF, "r") as f:
```

Comment:
In INFRASIM_TEMP_CONF, it maintains a jinja template, please check data/conf/infrasim.yml for detail.
This function create_infrasim_conf() read this template

    /usr/local/etc/infrasim/conf/infrasim.yml

and render the template with some runtime data, for example, network name, then write it to 

    /etc/infrasim/